docs/help/Getting-Started.md

```diff
@@ -74,8 +76,7 @@ build/x86_64-linux/debug/bin/iotjs tools/check_test.js -- start-from=test_consol
 ##### To write a test case
 
 1. Write a test case inside test directory.
-2. List up a test case in **test/testsets.js**. (this procedure will be removed after ``fs.readDir[Sync]`` is implemented.)
-3. Set attributes on the test case if it needs in **attrs.js** where the directory of your test case belongs.
+2. List up the test case in **test/testsets.json**, and set attributes on the test case if it needs.
```

Comment:
Would you please add detailed description for each `resources`, `run_fail` and `run_pass` ?